[32mINFO    [0m Pageobjects.Basepage:Basepage.py:31 Clicking on element: LANGUAGE_OPTION_XPATH
[32mINFO    [0m Pageobjects.Basepage:Basepage.py:31 Clicking on element: CONTINUE_BUTTON_ID
[33mWARNING [0m urllib3.connectionpool:connectionpool.py:812 Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10bb2df90>: Failed to establish a new connection: [Errno 61] Connection refused')': /wd/hub/session/074bc17b-413d-40c2-b101-2dc96034bb5d
[33mWARNING [0m urllib3.connectionpool:connectionpool.py:812 Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10bb2e050>: Failed to establish a new connection: [Errno 61] Connection refused')': /wd/hub/session/074bc17b-413d-40c2-b101-2dc96034bb5d
[33mWARNING [0m urllib3.connectionpool:connectionpool.py:812 Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x10bb2dd50>: Failed to establish a new connection: [Errno 61] Connection refused')': /wd/hub/session/074bc17b-413d-40c2-b101-2dc96034bb5d